src/inc/redirect/Mlp_Language_Negotiation.php

```diff
@@ -53,12 +53,13 @@ public function __construct(
 	}
 
 	/**
+	 * @param array $args
+	 *
 	 * @return array
 	 */
-	public function get_redirect_match() {
-
+	public function get_redirect_match( $args = array() ) {
 		$translations = $this->language_api->get_translations(
-			array ( 'include_base' => TRUE )
+			wp_parse_args( $args, array( 'include_base' => true ) )
```

Comment:
Since we don't allow for a query string, please use `array_merge()` here, as it is a core PHP function and thus doesn't need to be mocked in unit tests. Also, the uppercase `TRUE` is actually correct - well, in MultilingualPress v2, at least. 😀